### PR TITLE
Minor README updates

### PR DESCRIPTION
**Pull request by thomashoneyman**
This PR updates the README to:

1. Emphasize spago and de-emphasize psc-package, in accordance with their relative use in the community
2. Re-word a few things to emphasize salient parts of the package sets
3. Remove the banner image, as it takes up quite a bit of space and isn't obviously relevant to package sets (the Spago image, in contrast, is much nicer).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,36 @@
...
-A curated list of PureScript packages for Psc-Package and Spago
-
-![](https://i.imgur.com/roCuNQ9.png)
-
+A curated list of PureScript packages for the `spago` and `psc-package` package managers.
 
 - [What is a package-set?](#what-is-a-package-set)
 - [Add your package](#add-your-package)
 - [How do I use package-sets with `psc-package`?](#how-do-i-use-package-sets-with-psc-package)
 - [How do I use package-sets with `spago`?](#how-do-i-use-package-sets-with-spago)
```
Comment (JordanMartinez) [inline README.md:10]: Should this line appear before the `psc-package` one?

File: README.md [hunk 2]
```diff
@@ -2,31 +2,36 @@
...
-In general we welcome all packages, provided that they follow some guidelines defined in the [contributing guide](CONTRIBUTING.md).
+[`spago`][spago] is a package manager and build tool for PureScript. It is very similar to `psc-package`, and the main differences are:
+- it uses Dhall for its configuration (instead of `json` as `psc-package` does)
+- it supports package overrides and additions directly in the project configuration
+- it supports local dependencies (think `bower link`)
+
+With `spago` the package-set address is specified in the `upstream` variable of your local `packages.dhall`, which will usually import a remote `packages.dhall`, e.g. the one from this repo.
 
-The linked document also contains instructions on how to add new packages to the set, and information on versioning and related policies.
+You can change the package-set version you are using by just pointing the `upstream` to a different location, and running `spago freeze` afterwards (see [its readme][spago] for more info about this)
```
Comment (JordanMartinez) [inline README.md:34]: I think this can be better summarized as "run `spago upgrade-set --tag psc-...` since then spago freezes it in the same process.

**Review by f-f — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (thomashoneyman) [inline README.md:10]:
```suggestion
- [How do I use package-sets with `spago`?](#how-do-i-use-package-sets-with-spago)
- [How do I use package-sets with `psc-package`?](#how-do-i-use-package-sets-with-psc-package)
```

File: README.md (hunk 2 above)
Comment (thomashoneyman) [inline README.md:34]:
```suggestion
You can change the package-set version you are using by running `spago upgrade-set`, or if you want to point to a particular version, `spago upgrade-set --tag <tag>`.
```

**Pull request merged**